pkg/controller/healthz.go

```diff
@@ -0,0 +1,43 @@
+package controller
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/logging"
+	"github.com/google/exposure-notifications-verification-server/pkg/render"
+
+	"golang.org/x/time/rate"
+)
+
+var (
+	rl = rate.NewLimiter(rate.Every(time.Minute), 1)
+)
+
+func HandleHealthz(h *render.Renderer, cfg *database.Config) http.Handler {
```

Comment:
This should accept a context and pull the logger from the context. The logger isn't stored on the request context, it's stored on the handler context.

Reply:
Oh that's interesting, and surprising. Not what I would have expected, but changed.